<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1961</width>
    <height>1067</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QChartView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>271</height>
     </size>
    </property>
   </widget>
   <widget class="QChartView" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>439</y>
      <width>1371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>261</height>
     </size>
    </property>
   </widget>
   <widget class="QChartView" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>160</y>
      <width>581</width>
      <height>601</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>261</height>
     </size>
    </property>
   </widget>
   <widget class="QChartView" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>1371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>261</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_NengLi">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>760</y>
      <width>171</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>811</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>从</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>分钟取样</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>开始，间隔</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2020</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="date">
          <date>
           <year>2020</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>能力图数据</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="getSQLit">
         <property name="text">
          <string>指定时间开始查询</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="zhengya">
         <property name="text">
          <string>正压开启压力分析</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="fuya">
         <property name="text">
          <string>负压开启压力分析</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1961</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
